coding-intro/coding-intro-core/core-concepts/programs-and-data.md

```diff
@@ -20,22 +20,23 @@ Let's quickly introduce the building blocks we'll cover in this course.
 
 ### Program
 
-Even though they're super fast, computers are annoyingly literal - you have to spell out everything for them. 
+Today you woke up with a plan, a to-do list, which is essentially your program for the day. Similarly, we give the computers a set of commands to carry out, one after the other. That too is known as a program.
 
-> 💡 Check out the *Learn more* section for a realistic (and funny) example of this 🤓
+The only difference being that computers require specific instructions. (Don't tell anyone we said this, but machines are actually pretty dumb… sssh!)
 
-A program consists of one or more commands we're asking the computer to do. It's a sequence of specific tasks. A list of TODOs for the computer.
+> 💡 Scroll down to see a hilarious real life example of this in the Learn more section 😆
 
-### Variables
 
-For a program to remember values, it needs a place to store them. 
+### Data: Variables
 
-This is where variables come in. They serve as storage containers for data.
+Data is information and information needs to be stored somewhere. That’s where variables come in. For instance, when you meet someone new you could store their `name = “Batman”` and `age = 24` as two separate pieces of information.
```

Comment:
why do your quotes look weird? :laughing: 
```suggestion
Data is information and information needs to be stored somewhere. That’s where variables come in. For instance, when you meet someone new you could store their `name = "Batman"` and `age = 24` as two separate pieces of information.
```

Reply:
I think I switched them to curly using Grammarly, that's why.